MAUI/Calendar/customizations.md

```diff
@@ -0,0 +1,249 @@
+
+{% tabs %}
+{% highlight xaml tabtitle="MainPage.xaml" %}
+
+<calendar:SfCalendar  x:Name="Calendar" 
+                        View="Month"
+                        Background="PaleGreen">
+            <Calendar:SfCalendar.MonthView>
+                <Calendar:CalendarMonthView
+                TodayBackground="Pink" DisabledDatesBackground="Grey" SpecialDatesBackground="#FFEFD2" WeekendDatesBackground="#E2F9F3"/>
+            </Calendar:SfCalendar.MonthView>
+</calendar:SfCalendar>
+
+{% endhighlight %}
+{% highlight c# tabtitle="MainPage.xaml.cs" %}
+
+        CalendarTextStyle textStyle = new CalendarTextStyle()
+        {
+            TextColor = Colors.Black,
+            FontSize = 12,
+        };
+
+        this.Calendar.MinimumDate = DateTime.Now.AddDays(-15);
+        this.Calendar.MaximumDate = DateTime.Now.AddDays(20);
+        this.Calendar.EnablePastDates = true;
+        this.Calendar.SelectableDayPredicate = (date) =>
+        {
+            if (date.Date == DateTime.Now.AddDays(3).Date || date.Date == DateTime.Now.AddDays(6).Date || date.Date == DateTime.Now.AddDays(-3).Date || date.Date == DateTime.Now.AddDays(-6).Date)
+            {
+                return false;
+            }
+
+            return true;
+        };
+
+        this.Calendar.MonthView.SpecialDates = new List<DateTime>
+        {
+            DateTime.Now.AddDays(2),
+            DateTime.Now.AddDays(-2),
+        };
+
+        this.Calendar.MonthView.WeekendDays = new List<DayOfWeek>
+        {
+            DayOfWeek.Sunday,
+            DayOfWeek.Saturday,
+        };
+
+        this.calendar.Background = Colors.PaleGreen.WithAlpha(0.3f);
+        this.Calendar.MonthView.TextStyle = textStyle;
+        this.calendar.MonthView.TodayBackground = Colors.Pink;
+        this.Calendar.MonthView.TodayTextStyle = textStyle;
+        this.Calendar.ShowTrailingAndLeadingDates = true;
+        this.Calendar.MonthView.TrailingLeadingDatesBackground = Colors.Red.WithAlpha(0.3f);
+        this.Calendar.MonthView.TrailingLeadingDatesTextStyle = textStyle;
+        this.calendar.MonthView.DisabledDatesBackground = Colors.Grey.WithAlpha(0.3f);
+        this.Calendar.MonthView.DisabledDatesTextStyle = textStyle;
+        this.calendar.MonthView.SpecialDatesBackground = Color.FromArgb("#FFEFD2");
+        this.calendar.MonthView.SpecialDatesTextStyle = textStyle;
+        this.calendar.MonthView.WeekendDatesBackground = Color.FromArgb("#E2F9F3");
+        this.calendar.MonthView.WeekendDatesTextStyle = textStyle;
+
+{% endhighlight %}
+{% endtabs %}
+
+![Month view Customization in .NET MAUI Calendar.](images/customization/maui-month-view-customization.png)
+
+>**NOTE**
+* The Background color and text style will be applied based on the following order: selectableDayPredicate dates, special dates, disable dates, today date, trailingLeading dates, and weekend dates.
+
+## Month View Header format
```

Comment:
Specify this view header format, and header format under header topic

Reply:
Content modified